build.go

```diff
@@ -135,6 +137,10 @@ func build() {
 	runCmd("go", nil, "build", "-o", "build/octant", GO_FLAGS, "-v", "./cmd/octant")
 }
 
+func runDev() {
+	runCmd("build/octant", nil)
```

Comment:
Can you make this check for the existence of `build/octant` and run `build` if it isn't find?

Reply:
I updated the code and split the commits. Now `newCmd` will check for the existence of the command first.